src/Processors/Formats/Impl/NativeORCBlockInputFormat.cpp

```diff
@@ -57,13 +58,26 @@ class MemoryPool : public orc::MemoryPool
 public:
     char * malloc(uint64_t size) override
     {
-        auto * ptr = ::malloc(size);
+        void * ptr = ::malloc(size);
```

Comment:
Why we do not use `new` for which we always do tracking (IIRC it's overloaded)?

Reply:
We may use `new` here, but, we don't have proper type due to the abstraction, so I think it is OK to keep `malloc`